mappings/net/minecraft/item/SuspiciousStewEffectContainer.mapping

```diff
@@ -1,4 +1,15 @@
 CLASS net/minecraft/unmapped/C_noldxssx net/minecraft/item/SuspiciousStewEffectContainer
-	METHOD m_gaivnnnr getAll ()Ljava/util/List;
+	METHOD m_gaivnnnr getAllContainers ()Ljava/util/List;
+	METHOD m_tknkyyvb getAllEffects ()Ljava/util/List;
 	METHOD m_zuxvbyei getContainerOrNull (Lnet/minecraft/unmapped/C_gmbqjnle;)Lnet/minecraft/unmapped/C_noldxssx;
```

Comment:
```suggestion
	METHOD m_zuxvbyei getContainer (Lnet/minecraft/unmapped/C_gmbqjnle;)Lnet/minecraft/unmapped/C_noldxssx;
```
same as other `OrNull`, might as well change this one too if there's agreement that it's not needed